src/scenes/NurseryRouter/NurseryReassignmentView.tsx

```diff
@@ -233,7 +233,7 @@ export default function NurseryReassignmentView(): JSX.Element {
             id='back'
             to={`${APP_PATHS.NURSERY_WITHDRAWALS}?tab=withdrawal_history`}
             className={classes.backToWithdrawals}
-            name={strings.WITHDRAWAL_LOG}
```

Comment:
Should we remove the string from the string file too?

Reply:
I haven't been removing unused strings because they get added back automatically unless they're also removed from Phrase. Just requested access to Phrase though so will remove once I get access.